app/livechat/server/startup.js

```diff
@@ -96,4 +97,13 @@ Meteor.startup(() => {
 
 		monitor.start();
 	});
+
+	settings.get('Livechat_enable_business_hours', (key, value) => {
+		if (value) {
+			Promise.await(businessHourManager.openBusinessHoursIfNeeded());
+			return Promise.await(businessHourManager.createCronJobsForWorkHours());
+		}
+		businessHourManager.removeBusinessHoursFromAgents();
```

Comment:
This method is supposed to return a Promise, but it's not returning anything.
Should not it be:
```suggestion
		await businessHourManager.removeBusinessHoursFromAgents();
```

???